static/src/js/components/OrderProgressList/__tests__/OrderProgressItem.test.js

```diff
@@ -1,57 +1,115 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
 
-import { Badge, ProgressBar } from 'react-bootstrap'
+import { render, screen } from '@testing-library/react'
+
 import { OrderProgressItem } from '../OrderProgressItem'
 
-import { retrievalStatusPropsEsi } from '../../OrderStatus/__tests__/mocks'
+import {
+  retrievalStatusPropsEsi,
+  retrievalStatusPropsSwodlrOrder,
+  retrievalStatusPropsHarmonyOrder
+} from './mocks'
 
 beforeEach(() => {
   jest.clearAllMocks()
 })
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup(overrideProps) {
-  const order = retrievalStatusPropsEsi.retrieval.collections.byId[1].orders[0]
+const setup = (overrideProps) => {
+  let props = {}
 
-  const props = {
-    order,
+  props = {
     ...overrideProps
   }
 
-  const enzymeWrapper = shallow(<OrderProgressItem {...props} />)
+  render(<OrderProgressItem {...props} />)
 
   return {
-    enzymeWrapper,
     props
   }
 }
 
 describe('OrderProgressItem component', () => {
-  test('shows the correct order metadata', () => {
-    const { enzymeWrapper } = setup()
+  describe('Complete Swodlr Order', () => {
+    test('shows the correct order metadata', () => {
+      setup({
+        order: {
```

Comment:
```suggestion
        order: retrievalStatusPropsSwodlrOrder
```

All of your mocks used in this file have this chain of `.retrieval.collections.byId[1].orders[0]`.

Mock data should only include the actual data for the mock, everything else is unnecessary and can lead to confusion later.